pwndbg/commands/context.py

```diff
@@ -150,6 +152,87 @@ def contextoutput(section, path, clearing, banner="both", width=None):
                                     banner_top= banner in ["both", "top"],
                                     banner_bottom= banner in ["both", "bottom"])
 
+# Ghidra integration through radare2
+# This is not (sorry couldn't help it) "the yellow of the egg"
+# Using radare2 is suboptimal and will only be an intermediate step to have
+# ghidra decompiler within pwndbg.
+# But having ghidra in pwndgb is a lot more work as it requires quite some code and c/c++
+radare2 = {}
+
+config_context_ghidra = pwndbg.config.Parameter('context-ghidra',
+                                                  'always never if-no-source',
```

Comment:
According to the `Parameter` class this, the second argument is a default value. Shouldn't it be only `never` here? (see https://github.com/pwndbg/pwndbg/blob/dev/pwndbg/config.py#L113)

Reply:
Uh right, somehow I confused it with valid values.